app/proguard-rules.pro

```diff
@@ -20,6 +20,11 @@
 # debugging stack traces.
 -keepattributes SourceFile,LineNumberTable
 
+# required after agp 8 made r8 full mode default
```

Comment:
Also duplicated, the same classes are kept at the end of the file

Reply:
> Also duplicated, the same classes are kept at the end of the file

My bad, I deleted duplicated code in both file.